backend/pennclubs/settings/base.py

```diff
@@ -108,6 +108,8 @@
     {"NAME": "django.contrib.auth.password_validation.NumericPasswordValidator"},
 ]
 
+CORS_ALLOW_ALL_ORIGINS = True
+CORS_URLS_REGEX = r"^/api/.*$"
```

Comment:
If you haven't already, maybe double check with Armaan about allowing CORS to the whole API. He might prefer it to be restricted just to certain endpoints (like the member list).

Reply:
Messaged him